x-pack/legacy/plugins/uptime/index.ts

```diff
@@ -16,6 +17,13 @@ export const uptime = (kibana: any) =>
     id: PLUGIN.ID,
     publicDir: resolve(__dirname, 'public'),
     require: ['kibana', 'elasticsearch', 'xpack_main'],
+    config(Joi: any) {
+      return Joi.object({
+        enabled: Joi.boolean().default(true),
+      })
```

Comment:
You will need to add the `indexPattern` property in the legacy configuration for now, as both plugins share the same NS